cli/templates/Python/src/protected_data.py

```diff
@@ -0,0 +1,41 @@
+# <<protectedData>>
```

Comment:
whole module wrapped in a feature tag, will be removed if the protectedData feature is not selected